src/dissect/dissect.c

```diff
@@ -140,6 +151,30 @@ static int parse_argv(int argc, char *argv[]) {
                                 return r;
                         break;
 
+                case ARG_ROOT_HASH_SIG:
+                        if (startswith(optarg, "base64:")) {
+                                char *value;
+                                void *p;
+                                size_t l;
+
+                                value = strchr(optarg, ':') + 1;
```

Comment:
handily, startswith() actually returns a pointer to the first character after the specified prefix. hence rework to: 

```c
case ARG_ROOT_HASH_SIG: {
        const char *e;
        if ((e = startswith(optarg, "base64:"))) {
                r = unbase64mem(e, strlen(e), …);
…
```

you get the idea

Reply:
neat, done